### PR TITLE
Release Hydra 1.0

### PR DESCRIPTION
**Pull request by omry**

### CODE REVIEW

File: website/blog/2020-08-31-Hydra_1.0.md [hunk 1]
```diff
@@ -0,0 +1,431 @@
...
+
+</details>
+
+<br/>
+
+### Breaking changes
+For most people, migrating from 0.11 to 1.0 will be smooth.
+However, you may have issues if you are relying on Python 2 support or have used internal APIs.
+- Hydra 1.0 drops support for [Python 2](https://www.python.org/doc/sunset-python-2/).
+- If you are relying on internal APIs Hydra 1.0 will likely break your code. Maintaining backward compatibility for internal APIs is not a goal.
```
Comment (shagunsodhani) [inline website/blog/2020-08-31-Hydra_1.0.md:57]: If you are relying on internal APIs, Hydra

File: website/blog/2020-08-31-Hydra_1.0.md [hunk 2]
```diff
@@ -0,0 +1,431 @@
...
+- Hydra 0.11 Configs pickled and stored will not unpickle with Hydra 1.0 due to internal changes in OmegaConf.
+
+There are multiple new deprecation warnings, each with a link to mini migration guide.
```
Comment (shagunsodhani) [inline website/blog/2020-08-31-Hydra_1.0.md:60]: link to a mini migration guide.

File: website/blog/2020-08-31-Hydra_1.0.md [hunk 3]
```diff
@@ -0,0 +1,431 @@
...
+
+## Structured Configs
+Structured Configs is a new powerful feature that enables strongly typed configs
```
Comment (shagunsodhani) [inline website/blog/2020-08-31-Hydra_1.0.md:63]: Structured Config

File: website/blog/2020-08-31-Hydra_1.0.md [hunk 4]
```diff
@@ -0,0 +1,431 @@
...
+    port: 3306
+```
+</div></div>
+
+Learn more at [Advanced/Overriding packages](/docs/next/advanced/overriding_packages).
+
+## Command line grammar (Overrides grammar)
+Hydra 1.0 introduces a proper parser and grammar for the command line syntax, supporting a multitude of new use cases.
+
+A short list of examples:
```
Comment (shagunsodhani) [inline website/blog/2020-08-31-Hydra_1.0.md]: A shortlist of examples:

File: website/blog/2020-08-31-Hydra_1.0.md (hunk 2 above)
Comment (omry) [reply website/blog/2020-08-31-Hydra_1.0.md:60]: There are 4 different pages. and no way to link to all of them.

File: website/blog/2020-08-31-Hydra_1.0.md (hunk 4 above)
Comment (omry) [reply website/blog/2020-08-31-Hydra_1.0.md]: already outdated.

File: website/blog/2020-08-31-Hydra_1.0.md [hunk 5]
```diff
@@ -0,0 +1,431 @@
...
+
+
+## Compose API improvements
+The experimental Compose API is maturing. It is now possible to initialize Hydra in one of 3 ways:
+- `initialize()`: Initialize with a config path relative to the caller
+- `initialize_config_module()` : Initialize with config_module (absolute)
+- `initialize_config_dir()` : Initialize with a config_dir on the file system (absolute)
+
+All initialization methods can be used to initialize Hydra globally or as a context.  
+Learn more at [Experimental/Compose API](/docs/next/experimental/compose_api).
```
Comment (shagunsodhani) [inline website/blog/2020-08-31-Hydra_1.0.md:319]: Just highlighting "Experimental/Compose API". Maybe you want it to be "Compose API"

File: website/blog/2020-08-31-Hydra_1.0.md (hunk 3 above)
Comment (omry) [reply website/blog/2020-08-31-Hydra_1.0.md:63]: I am actually not sure this is correct. The feature is called "Structured Configs".

File: website/blog/2020-08-31-Hydra_1.0.md [hunk 6]
```diff
@@ -0,0 +1,431 @@
...
+- The [Nevergrad Sweeper](/docs/next/plugins/nevergrad_sweeper) brings the power of [Nevergrad](https://github.com/facebookresearch/nevergrad) to your Hydra app 
+
+Note that both sweepers are still in early stages and some changes are expected soon. We welcome any feedback though.
+
+### Tab completion
+In addition to Bash, Hydra now supports [Fish](https://fishshell.com/) shell Tab Completion.
+
+
+## Misc improvements
+- Override the config_path and config_name from the command line with --config-name and --config-path
```
Comment (shagunsodhani) [inline website/blog/2020-08-31-Hydra_1.0.md:425]: `config_path` and `config_name`

Comment (omry) [reply website/blog/2020-08-31-Hydra_1.0.md:425]: ?

File: website/blog/2020-08-31-Hydra_1.0.md (hunk 2 above)
Comment (shagunsodhani) [reply website/blog/2020-08-31-Hydra_1.0.md:60]: oops I meant "link to mini migration guide" -> "link to a mini migration guide"

**Pull request merged**